src/Omnipay/Komerci/BaseRequest.php

```diff
@@ -83,6 +83,21 @@ public function setNumAutor($value)
         return $this->setParameter('numautor', $value);
     }
 
+    public function getDate()
+    {
+        return $this->getParameter('date');
+    }
+
+    public function getFormattedDate()
+    {
+        return date('Ymd', strtotime($this->getParameter('date')));
```

Comment:
Se o parametro 'date' for vazio, assumir então a data de hoje. Isso garante o funcionamento de quem já possui versões implementadas desta biblioteca.